samples/README.rst

```diff
@@ -0,0 +1,206 @@
+.. This file is automatically generated. Do not edit this file directly.
```

Comment:
It seems like previously folks could edit the readme directly in the samples - how will that work with this being autogenerated?

Reply:
We have this problem with all the template files (noxfile.py, kokoro files). If someone submits a PR with an edit I ask them to make the change to the template source in synthtool.

If a change does sneak in, it'll be reverted in the next Autosynth PR.